drivers/video/video.c

```diff
@@ -1172,17 +1195,63 @@ static int video_reqbufs(FAR struct video_mng_s         *vmng,
     }
   else
     {
+      if (reqbufs->count > CONFIG_VIDEO_REQBUFS_COUNT_MAX)
+        {
+          reqbufs->count = CONFIG_VIDEO_REQBUFS_COUNT_MAX;
+        }
+
       video_framebuff_change_mode(&type_inf->bufinf, reqbufs->mode);
 
       ret = video_framebuff_realloc_container(&type_inf->bufinf,
```

Comment:
let's remove the count check in video_framebuff_realloc_container and V4L2_REQBUFS_COUNT_MAX

Reply:
And also remove everything related to V4L2_REQBUFS_COUNT_MAX?

Reply:
done